### PR TITLE
test-vf-vf-fwd.sh: Bring vfs up before configuring them

### PR DESCRIPTION
**Pull request by ZiyadAtiyyeh**
Signed-off-by: Ziyad Atiyyeh <ziyadat@mellanox.com>

### CODE REVIEW

File: test-vf-vf-fwd.sh [hunk 1]
```diff
@@ -76,6 +76,8 @@ else
 fi
 
 start_clean_openvswitch
+ifconfig $VF up
+ifconfig $VF2 up
```
Comment (roidayan) [inline test-vf-vf-fwd.sh]: this is not the correct place. we have function config_vf() to config each VF. and i think config_vf already brings the VF up.

Comment (ZiyadAtiyyeh) [reply test-vf-vf-fwd.sh]: The config_vf() checks for the vf's existence before bringing them up and in our case here we got errors. So, if you want i can move "ifconfig $VF up" and "ifconfig $VF2 up" to the beginning of config_vf(), what do you think?

Comment (roidayan) [reply test-vf-vf-fwd.sh]: of course it checks for existence else there is nothing to bring ip

Comment (roidayan) [reply test-vf-vf-fwd.sh]: i think this pull request is incorrect and doesn't explain any issue. if /sys/class/net/DEV doesn't exists there is no reason to execute ifconfig DEV up (as it doesn't exists). so your explanation doesn't make sense for me.

**Pull request closed without merging**